src/content/post/drush-make-gitignore.md

```diff
@@ -0,0 +1,52 @@
++++
+date        = "2016-04-08T01:00:00-04:00"
+title       = "Drush Make overwrites your custom .gitignore file"
+description = "An investigation about an undesired behavior: bug or feature?"
+tags        = [ "development", "drupal", "drush"  ]
+topics      = [ "Drupal" ]
+slug        = "drush-make-gitignore"
+author      = "Marcello Testi"
++++
+
+# Managing a customized .gitignore file across drush make builds
+
+## The problem
+
+**[Drush](https://github.com/drush-ops/drush) 8** overwrites the .gitignore file after a successful makefile execution.
+
+This affects complex projects where multiple dependency sources are present, and where other reasons might require a significant override of the **.gitignore** file provided by Drupal, which ends up diffed, causing `git status` to produce an ugly output, until the committed .gitignore is restored.
+
+## Origin of the problem
+
+In recent upgrades, the `--overwrite` option has been added to Drush in order to force the overwrite of existing folders (the command docs don't  mention files, althought some file reference is present in the code comments), while the default behavior is to *merge*, which means keeping existing folder content. This option has only an impact on **directories** inside the project's root and doesn't affect the files in the root.
+
+## How the problem was investigated
+
+ * I created a couple of docker containers based on SparkFabrik image ready to be used for PHP/Drupal development: https://hub.docker.com/r/sparkfabrik/docker-php-base-image/
```

Comment:
I'd rephrase this a bit as:

`I created a couple of docker containers based on [SparkFabrik base PHP image](https://hub.docker.com/r/sparkfabrik/docker-php-base-image/) well suitable for PHP/Drupal development.`